src/modules/components/proposal/proposalActions/utils/proposalActionsUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { ProposalActionWithdrawToken } from '../actions/proposalActionWithdrawToken/proposalActionWithdrawToken';
```

Comment:
As we won't have more utility files, I'd move the utils and test files to the `/proposalActions` folder directly